### ISSUE TITLE
[Accessibility]A11y_VSCode_ScreenReaderOptimized_Keyboard:"Screen Reader Optimized" button is not accessible by keyboard

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52200]
Environment Details:
VSCode Version : 1.24.1
OS Version : Win10

Repro Steps:
1. Launch VS Code.
2. Navigate to File->select New File->Navigate to Status Bar-> "Screen Reader Optimized" button.
3. Check the keyboard navigation.

Actual:
"Screen Reader Optimized" button is not accessible by keyboard.it is only accessible through mouse also there is no shortcuts provided access this.

Expected:

"Screen Reader Optimized" button should accessible through keyboard.

Recommendations:
Refer below link which is repository of bug fixes code snippets:
[https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx](url)

User Impact:
The keyboard user will not be able to access the control.
MAS Reference:
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}](url)

Does this issue occur when all extensions are disabled?: Yes
Attachment For Reference:

**Comment by isidorn** [comment]
Dup of https://github.com/Microsoft/vscode/issues/51977
